server/routes/friendsListRouter.js

```diff
@@ -0,0 +1,54 @@
+const router = require('express').Router();
+const User = require('../models/userModel');
+const FriendsList = require('../models/friendsListModel');
+
+router.post('/upload', async (req, res) => {
```

Comment:
```suggestion
router.post('/', async (req, res) => {
```

A route called `/friendLists` with method `POST` should signify that you're creating a new friend list